benchmarks/datasets.py

```diff
@@ -0,0 +1,28 @@
+from torchvision import transforms, datasets
+import torch
+
+def transform(img):
+    t= transforms.Compose([
+        transforms.ToPILImage(),
+        transforms.Resize(size=(100,100)),
+        transforms.ToTensor()]
+    )
+    return t(img)
+
+def str_to_list(str):
+    l = []
+    for char in str:
+        l.append(int(char))
+    return l
+
+def prepare_gtsrb_datapipe(batch_size, device, dp):
+    # Filter out bounding box and path to image
+    dp = dp.map(lambda sample : {"image" : sample["image"], "label" : sample["label"]})
+
+    # Apply image preprocessing
+    dp = dp.map(lambda sample : transform(sample.decode().to(torch.device(device))), input_col="image")
+    dp = dp.map(lambda sample : torch.tensor(str_to_list(sample.to_categories())).to(torch.device(device)), input_col="label")
```

Comment:
I wonder if the usage of `lambda` will lead to issues with multiprocessing if `dill` isn't installed. Or is `dill` always installed here?

Reply:
I can just add dill to the list of dependencies in my own requirements.txt